### PR TITLE
Add timesColumnWidth prop

### LINKED ISSUE #251: Feature request: Cannot change <Title> width
**Describe the feature you'd like**
I would like to change the width of the Title header and Times column.

in utils.js
`const TIMES_WIDTH_PERCENTAGE = 18;
const PAGE_WIDTH_PERCENTAGE = (100 - TIMES_WIDTH_PERCENTAGE) / 100;`
It's fixed at 18%. cannot change.

### PR DESCRIPTION
**Pull request by pdpino**
Fixes #251

### CODE REVIEW

File: example/App.js [hunk 1]
```diff
@@ -202,6 +202,7 @@ const App = ({}) => {
           startHour={15}
           fixedHorizontally={showFixedComponent}
           showTitle={!showFixedComponent}
+          timesColumnWidth={0.2}
```
Comment (hoangnm) [inline example/App.js:205]: Should the name be clearer?

Comment (pdpino) [reply example/App.js:205]: Sure it can be better. Is the best I came up with 😅

Do u have something in mind? What u think is not clear from the name?

Comment (hoangnm) [reply example/App.js:205]: I meant the name of the prop `timesColumnWidth`, it makes me think I should put the fixed width value, but actually, it's the ratio

Comment (hoangnm) [reply example/App.js:205]: I guess `timesColumnWidthRatio` could be the name, WDYT, @pdpino ?

Comment (pdpino) [reply example/App.js:205]: It can be provided either as a ratio (`0 < value < 1`) or as fixed width value (`1 <= value`)

Comment (hoangnm) [reply example/App.js:205]: I see, I missed this line, https://github.com/hoangnm/react-native-week-view/pull/253/files#diff-accc88f2dd24cd8f77ae72d8b3871d3eb26c0b086886fb5e8ba03fc65200dcfaR108, but what if the value is over 100?

Comment (pdpino) [reply example/App.js:205]: (sorry the late reply)

The code is wrong, is not doing what I intended. Good catch.

I'll fix this asap and let you know when it's ready

Comment (hoangnm) [reply example/App.js:205]: It looks great!

**Comment by hoangnm** [comment]
Thanks again, LGTM.

**Pull request merged**